dev-portal/src/components/SwaggerUiLayout.jsx

```diff
@@ -56,13 +56,13 @@ const SubscriptionButtons = observer(class SubscriptionButtons extends React.Com
   render() {
     const { api } = store
     return (
-      (api && isAuthenticated()) ? (
+      (api && isAuthenticated()) ? !api.generic ? (
```

Comment:
Is this... 3 nested ternaries? Lines 59-60? How many states do we really have? Do we need to represent 6 states?

Reply:
Almost.

- no api
- api and not signed in
- api and signed in and generic
- api and signed in and apigateway and subscribed
- api and signed in and apigateway and unsubscribed